src/main/java/RedBot/Bot.java

```diff
@@ -138,7 +146,7 @@ else if(message[1].equalsIgnoreCase("random")){ //Completely random quote
                 }
                 break;
                 
-            case "hentai": // TODO Optimise this train wreck
+            case "hentai": // TODO Optimise this train wreck            Good fucking luck lmao -Egg
```

Comment:
I'd appreciate it if you actually tried making it work

Reply:
idk how to 😔